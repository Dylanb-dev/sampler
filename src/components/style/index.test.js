/* eslint-disable */

import React from 'react'
import renderer from 'react-test-renderer'

import {
  FlexBetween,
  FlexVerticalCenter,
  Flex,
  ColumnSection,
  AppContainer
} from './'

test('FlexBetween renders correctly', () => {
  const tree = renderer
    .create(
      <FlexBetween>
        <div />
        <div />
      </FlexBetween>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

test('FlexVerticalCenter renders correctly', () => {
  const tree = renderer
    .create(
      <FlexVerticalCenter>
        <div />
        <div />
      </FlexVerticalCenter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

test('Flex renders correctly', () => {
  const tree = renderer
    .create(
      <Flex>
        <div />
        <div />
      </Flex>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

test('ColumnSection renders correctly', () => {
  const tree = renderer
    .create(
      <ColumnSection>
        <div />
        <div />
      </ColumnSection>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

test('AppContainer renders correctly', () => {
  const tree = renderer
    .create(
      <AppContainer>
        <div />
        <div />
      </AppContainer>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
