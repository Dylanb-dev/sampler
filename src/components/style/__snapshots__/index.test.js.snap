// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "linear-gradient(#146E14,#181818)",
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
      "maxHeight": "100vh",
      "maxWidth": "100vw",
      "minHeight": "100vh",
      "minWidth": "100vw",
    }
  }
>
  <div />
  <div />
</div>
`;

exports[`ColumnSection renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <div />
  <div />
</div>
`;

exports[`Flex renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "space-around",
      "width": "100%",
    }
  }
>
  <div />
  <div />
</div>
`;

exports[`FlexBetween renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "height": "100vh",
      "justifyContent": "space-between",
      "width": "100%",
    }
  }
>
  <div />
  <div />
</div>
`;

exports[`FlexVerticalCenter renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <div />
  <div />
</div>
`;
